src/openlcb/StreamSender.hxx

```diff
@@ -94,20 +93,26 @@ public:
     /// Initiates using the stream sender. May be called only on idle stream
     /// senders.
     ///
+    /// @param src Source virtual node on the local interface.
     /// @param dst Destination node ID to send the stream to.
     /// @param source_stream_id 8-bit stream ID to use on the this (the source)
     /// side.
+    /// @param dst_stream_id 8-bit stream ID to use on the this (the source)
```

Comment:
is this description correct? Shouldn't it to be the remote side?

Reply:
fixed